spec.html

```diff
@@ -3881,6 +3904,17 @@ <h1>InitializeReferencedBinding ( _V_, _W_ )</h1>
           1. Return _base_.InitializeBinding(_V_.[[ReferencedName]], _W_).
         </emu-alg>
       </emu-clause>
+
+      <emu-clause id="sec-makeprivatereference" aoid="MakePrivateReference">
+        <h1>MakePrivateReference ( _baseValue_, _privateIdentifier_ )</h1>
+        <emu-alg>
+          1. Let _env_ be the running execution context's PrivateEnvironment.
+          1. Let _privateNameBinding_ be ? ResolveBinding(_privateIdentifier_, _env_).
```

Comment:
I believe this is infallible. Here's a patch to switch from "? ResolveBinding" to "! ResolveBinding" https://github.com/Ms2ger/ecma262/pull/1

Reply:
This makes sense. I'm adding this manually, since my rebase yesterday broke the PR

Reply:
No longer relevant as of https://github.com/tc39/ecma262/pull/1668/commits/fa2a42430b32a0f074906f68d6d078fd09d8d8cc.